planos/005-exploration-gabwindow/feedback-gabwindow/builds/fase1/activate-gab-helper.js

```
@@ -0,0 +1,20 @@
+"use strict";
+const fs = require("fs");
+const file = "frontend/js/plugins.js";
+const source = fs.readFileSync(file, "utf8");
+const prefix = "var $plugins =";
+const marker = source.indexOf(prefix);
+const start = source.indexOf("[", marker);
+const end = source.lastIndexOf("]");
+if (marker < 0 || start < 0 || end < start) throw new Error("Unexpected plugins.js envelope");
+const plugins = JSON.parse(source.slice(start, end + 1));
+if (plugins.some(plugin => plugin.name === "Coreto_GabWindowDefaults")) throw new Error("Helper already active");
+const gabIndex = plugins.findIndex(plugin => plugin.name === "VisuMZ_4_GabWindow");
```

ESLint

[error] Replace `"VisuMZ_4_GabWindow"` with `'VisuMZ_4_GabWindow'`

(prettier/prettier)